<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinishPanelClass</class>
 <widget class="QWidget" name="FinishPanelClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FinishPanel</string>
  </property>
  <widget class="QLabel" name="winnerImg">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>141</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>没有</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TankWarMapEditor">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>胜者是</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="interestringLabel1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRestart">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>530</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>重新开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReturn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>530</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回主菜单</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
